tests/test_trackingstore.py

```diff
@@ -0,0 +1,174 @@
+# Copyright 2019 Faculty Science Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+from datetime import datetime
+from uuid import uuid4
+
+import faculty
+from faculty.clients.base import HttpError
+from faculty.clients.experiment import Experiment
+from mlflow.entities import Experiment as MLExperiment, LifecycleStage
+from mlflow.exceptions import MlflowException
+import pytest
+from pytz import UTC
+
+from mlflow_faculty.trackingstore import FacultyRestStore
+
+
+PROJECT_ID = uuid4()
+STORE_URI = "faculty:{}".format(PROJECT_ID)
+
+EXPERIMENT_ID = 12
+CREATED_AT = datetime.now(tz=UTC)
+LAST_UPDATED_AT = CREATED_AT
+
+NAME = "experiment name"
+ARTIFACT_LOCATION = "scheme://artifact-location"
+
+MLFLOW_EXPERIMENT = MLExperiment(
+    EXPERIMENT_ID, NAME, ARTIFACT_LOCATION, LifecycleStage.ACTIVE
+)
+
+FACULTY_EXPERIMENT = Experiment(
+    id=EXPERIMENT_ID,
+    name=NAME,
+    description="not used",
+    artifact_location=ARTIFACT_LOCATION,
+    created_at=datetime.now(tz=UTC),
+    last_updated_at=datetime.now(tz=UTC),
+    deleted_at=None,
+)
+
+
+def experiments_equal(one, two):
+    return (
+        one.experiment_id == two.experiment_id
+        and one.name == two.name
+        and one.artifact_location == two.artifact_location
+        and one.lifecycle_stage == two.lifecycle_stage
+    )
+
+
+@pytest.mark.parametrize(
+    "store_uri",
+    [
+        STORE_URI,
+        "faculty:/{}".format(PROJECT_ID),
+        "faculty:///{}".format(PROJECT_ID),
+        "faculty:///{}/".format(PROJECT_ID),
+    ],
+)
+def test_init(mocker, store_uri):
+    mocker.patch("faculty.client")
+
+    store = FacultyRestStore(store_uri)
+
+    assert store._project_id == PROJECT_ID
+    faculty.client.assert_called_once_with("experiment")
+
+
+@pytest.mark.parametrize(
+    "store_uri",
+    [
+        "invalid-scheme://{}".format(PROJECT_ID),
+        "faculty://{}".format(PROJECT_ID),
```

Comment:
I feel like a lot of people are going to write it like this, the colon-slash-slash muscle memory is quite strong (and it took me a moment to realize why it is wrong). We might leave this for a later PR, but since we are not using netloc at all we could consider grabbing the project ID from path *or* netloc. @acroz?